src/components/Tooltip/styles.module.css

```diff
@@ -0,0 +1,64 @@
+.tooltip {
+  visibility: hidden;
+  width: max-content;
+  position: absolute;
+  top: 0;
+  left: 0;
+  padding: 8px 16px;
+  border-radius: 3px;
+  font-size: 90%;
+  pointer-events: none;
+  opacity: 0;
+  transition: opacity 0.3s ease-out;
+  will-change: opacity, visibility;
+}
+
+.arrow {
+  position: absolute;
+  background: #222;
+  width: 8px;
+  height: 8px;
+  transform: rotate(45deg);
+}
+
+.show {
+  visibility: visible;
+  opacity: 0.9;
+}
+
+/** Types variant **/
```

Comment:
I'd recommend moving all default color values to CSS constants in default styles CSS file:
```
:root {
  --dark-background: #222;
  --dark-text-color: white;

...
}
```

and use it:
```
background: var(--dark-background);
color: var(--dark-text-color);

...
```

This way there will be one place for color changes and adding variant colors in the future

Reply:
Great!

I like CSS vars, thanks for the suggestion!